controllers/articles.controller.js

```diff
@@ -21,4 +21,24 @@ exports.getAllArticles = (req, res, next) =>{
         res.status(200).send({articles})
     })
     .catch(next)
+}
+
+exports.patchArticleById = (req, res, next) => {
+    const { article_id } = req.params;
+    const { inc_votes } = req.body;
+    const articlePromises = []
+    
+    if (article_id) {
+        articlePromises.push(checkArticleIdExists(article_id))
```

Comment:
We shouldn't need to worry about a check exists function here, try passing the error query through to psql to see if it throws an error for you. We can also check for the empty rows like you have later for the 404.